packages/beacon-node/test/e2e/api/impl/beacon/state/endpoint.test.ts

```diff
@@ -0,0 +1,101 @@
+import {expect} from "chai";
+import {SLOTS_PER_EPOCH} from "@lodestar/params";
+import {createBeaconConfig} from "@lodestar/config";
+import {chainConfig as chainConfigDef} from "@lodestar/config/default";
+import {Api, ApiError, getClient} from "@lodestar/api";
+import {computeCommitteeCount} from "@lodestar/state-transition";
+import {LogLevel, testLogger} from "../../../../../utils/logger.js";
+import {getDevBeaconNode} from "../../../../../utils/node/beacon.js";
+import {BeaconNode} from "../../../../../../src/node/nodejs.js";
+
+describe("beacon state api", function () {
+  this.timeout("10 min");
```

Comment:
This is too aggressive, can we make it more reasonable?

Reply:
yeah, might be good idea to reduce that, just copied the value from https://github.com/ChainSafe/lodestar/blob/82914a1443fa1077d2a217cda9a1248e03084c46/packages/beacon-node/test/e2e/api/impl/lightclient/endpoint.test.ts#L18